docs/managing-files/integrity.rst

```diff
@@ -0,0 +1,62 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2017-2018 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Integrity checks
+================
+
+`Invenio-Files-REST <https://invenio-files-rest.readthedocs.io/>`_ provides
+Celery task-based mechanism for ensuring files integrity. It consists of two
+dedicated functions (Celery tasks) in :py:mod:`invenio_files_rest.tasks`.
+
+Main task ``invenio_files_rest.tasks.schedule_checksum_verification`` is to be
+periodically called through `celery beat <https://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#id1>`_
+and is responsible for querying files and scheduling a subtask
```

Comment:
Not sure we have to be so thorough about how celery beat works. We can simplify it , something like:

will be called periodically and will run integrity checks ``invenio_files_rest.tasks.verify_checksum`` on the files.